test/ComponentTests/PlatformServices.Desktop.Component.Tests/DesktopTestSourceHostTests.cs

```diff
@@ -77,19 +66,24 @@ public void ChildDomainResolutionPathsShouldHaveSearchDirectoriesSpecifiedInRuns
                 </MSTestV2>
              </RunSettings>";
 
-            this.testSourceHost = new TestSourceHost(this.testSource, this.GetMockedIRunSettings(runSettingxml).Object, null);
-
+            var testSource = GetTestAssemblyPath("DesktopTestProjectx86Debug.dll");
+            this.testSourceHost = new TestSourceHost(testSource, this.GetMockedIRunSettings(runSettingxml).Object, null);
             this.testSourceHost.SetupHost();
 
+            var assemblyResolution = "ComponentTests\\TestProjectForAssemblyResolution.dll";
+            var asm = Assembly.LoadFrom(assemblyResolution);
+            var type = asm.GetType("PlatformServices.Desktop.ComponentTests.TestProjectForAssemblyResolution");
+
             // Creating instance of TestProjectForAssemblyResolution should not throw.
             // It is present in  <Directory path = ".\ComponentTests" />  specified in runsettings
-            AppDomainUtilities.CreateInstance(this.testSourceHost.AppDomain, typeof(TestProjectForAssemblyResolution), null);
```

Comment:
Why is the type not available anymore here?

Reply:
We are not referencing that test project anymore. It causes package compatibility errors with PackageReference restore.